### DISCUSSION TITLE
Filtering results for dynamic date range?

### DISCUSSION THREAD

**Discussion by calebmil** [microsoft-search/pnp-modern-search#3769] [Q&A] [answered]
I have a content type with a date/time column.  I want to be able to filter my results web part to only show future results.  I tried adding the following KQL to my query but it blocked out all results.

`RefinableDate01 >= today`

The column in question has been crawled and mapped to a RefinableDate field.  RefinableDate01 has been included in the selected properties list.

Should I be taking a different approach?  Am I missing something syntactically?  Any help from the community would be appreciated!

**Comment by kasperbolarsen** [comment]
Can you share the mapping of the RefinableDate01 ? (screen shot)

**Reply by calebmil** [reply]
![image](https://github.com/microsoft-search/pnp-modern-search/assets/6432243/9df1c1f2-46b3-4cec-87e7-d54d92c13a11)

**Reply by kasperbolarsen** [reply, marked as answer]
It should work, I have a similar setup:
![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/08c95a2a-0c34-423e-9e3b-5b3d4042d8b8)


Have you checked that RefinableDate01 contains data? ( use SP Editor or SharePoint Search query tool )

**Reply by calebmil** [reply]
Oh heck!  It was showing as null in debug mode.  Even though I had done the mapping this morning (several hours ago), it looks like the index had not been refreshed.  I pushed a reindex request for the list in question and now am seeing results.  Sorry for the newb mistake! <embarrased>

**Reply by kasperbolarsen** [reply]
You are not the first nor the last to assume that the indexing must have happened. I have done that myself on multiple occasions :-)

**Reply by calebmil** [reply]
Thanks for being a sounding board and helping me find the error of my ways!
